src/components/Button.jsx

```diff
@@ -74,14 +74,32 @@ function Core({
 
   if (display === 'link') {
     roleStyles = {
+      textTransform: 'none',
       background: 'none!important',
       border: 'none',
       padding: '0!important',
```

Comment:
i'm fairly certain the importants are no longer necessary since we're not overriding anything anymore